contracts/MAHAX.sol

```diff
@@ -55,8 +54,8 @@ contract MAHAX is ReentrancyGuard, IVotingEscrow, Ownable {
 
   mapping(uint256 => uint256) public attachments;
   mapping(uint256 => bool) public voted;
+  mapping(uint256 => bool) public staked;
```

Comment:
we can't change datapoints of an already depolyed proxy contract. this variable needs to be stored elsewhere